solr/core/src/test/org/apache/solr/handler/component/DebugComponentTest.java

```diff
@@ -40,6 +40,18 @@ public static void beforeClass() throws Exception {
     assertU(adoc("id", "1", "title", "this is a title.", "inStock_b1", "true"));
     assertU(adoc("id", "2", "title", "this is another title.", "inStock_b1", "true"));
     assertU(adoc("id", "3", "title", "Mary had a little lamb.", "inStock_b1", "false"));
+    assertU(
+        adoc("id", "4", "title", "Vegeta is the prince of all saiyans.", "inStock_b1", "false"));
+    assertU(
+        adoc(
+            "id",
+            "5",
+            "title",
+            "Kakaroth aka Son Goku is a legendary saiyan warrior",
+            "inStock_b1",
+            "false"));
+    assertU(
+        adoc("id", "6", "title", "Gohan is the first saiyan-human hybrid.", "inStock_b1", "false"));
```

Comment:
as you can see @dsmiley here I added more docs, that's the reason they are now 6